department-of-veterans-affairs/abd-vro
# MCP-2657:  6602v2 assessment service update

## What was the problem?

6602 had fallen out of date with requirements and standards used to handle MAS data. 

Associated tickets or Slack threads:

- [MCP-2657](https://amida.atlassian.net/browse/MCP-2657)

## How does this fix it?[^1]

Apply standards for MAS data handling. Add schedular medication evidence within one year calculation. 

Pair PR with #1516

## How to test this PR
- Step 1 Unit tests
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
